docs/docs/configuration/index.md

```diff
@@ -3,7 +3,7 @@ id: index
 title: Configuration
 ---
 
-HassOS users can manage their configuration directly in the addon Configuration tab. For other installations, the default location for the config file is `/config/config.yml`. This can be overridden with the `CONFIG_FILE` environment variable. Camera specific ffmpeg parameters are documented [here](cameras.md).
+For all installations, including for HassOS, the default location for the config file is `/config/frigate.yml`. This can be overridden with the `CONFIG_FILE` environment variable. Camera specific ffmpeg parameters are documented [here](cameras.md).
```

Comment:
This isn't true for non-HassOS installations. It is `/config/config.yml` for everything else.

Reply:
Ah, understood. Thanks for the clarification. Made a new commit which I think is now correct. The main point is clarifying that HassOS configs are not defined in the Addon UI.